docs/protocol.md

```diff
@@ -215,6 +215,11 @@ size to 64KiB to allow better interleaving of frames across a shared TCP
 connection. In order to handle most operations implementations will need to
 decode some part of the payload.
 
+### reserved:1
+
+Unused space for future protocol revisions, padded for read alignment, which
```

Comment:
Saying that this was "for read alignment" is stretching / retconning a bit.  Read alignment wasn't at all a goal when we designed the header.  I think this is more leftover from we used to have a top-level flags single-byte field.

Also the most useful artifact of all 9 of these total reserved bytes is that they make the body start at offset 0x10 which wraps to lines cleanly in a standard 16-byte wide hex dump (e.g. our own tcap, or even just hexer dumping internally for any reason).